### ISSUE TITLE
Donate plugins repo to falcosecurity

### ISSUE THREAD

**Issue by leogr** [falcosecurity/evolution#62]
**Description of Request**

The following two projects are parts of the [libs plugin system](https://github.com/falcosecurity/falco/pull/1637) proposed by @ldegio: 

- ~https://github.com/leogr/plugins~ https://github.com/mstemm/plugins *this repo will be the place where various plugins will live; currently, it includes four examples*
- ~https://github.com/ldegio/libsinsp-plugin-sdk-go *an SDK for implementing plugins in Golang* N.B.: we should change the name of this repo~ (Moved to a separate issue https://github.com/falcosecurity/evolution/issues/101)

This request has to be considered as a dependant of:
- The proposal :point_right: https://github.com/falcosecurity/falco/pull/1637
- The proposed implementation :point_right: ~https://github.com/falcosecurity/libs/pull/33~ https://github.com/falcosecurity/libs/tree/new/plugin-system-api-additions, no PR yet.

**Motivation**

The plugin system for https://github.com/falcosecurity/libs is a new awesome feature under development. 
Basically, this new mechanism will allow plugging new data sources to `libscap` and `libsinsp` so that the library's consumers can use them. Thus paving the way to a lot of new possibilities, really. :partying_face: 

When the implementation becomes official, other fundamental pieces will be needed, since:
- plugins need a place and a registry (a unique ID is required for each plugin);
- to facilitate the development of plugins in Golang an SDK is needed (to avoid boilerplates).

Those repositories are WIPs that aims to address the above topics. The repos also reached a point where **feedback is needed**, and incubating them at some point would help to test-drive implementations.

**Comment by fntlnz** [comment]
This is a major change on how Falco and the libraries work, we need a proposal for this.

**Comment by leogr** [comment]
> This is a major change on how Falco and the libraries work, we need a proposal for the plugin system first, then we can discuss the incubation for the two repositories once the feature is official. I'm not ok with having the two repositories in before merging the feature.

I agree with that, so I have updated the description to make that clearer. My initial intention was more about receiving feedback, but it's totally right we first need that the proposal and the implementation get accepted :+1:

**Comment by mstemm** [comment]
We've been moving repos around a little bit while we finished the work, so I created a separate issue to cover just the plugin-sdk-go repository: https://github.com/falcosecurity/evolution/issues/101.

Let's let this issue cover the plugins themselves.

I updated the description to note that and updated the repo we'd like to donate to https://github.com/mstemm/plugins.

Please let us know what you think!
